dss/api/search.py

```diff
@@ -1,24 +1,19 @@
 import json
-import os
 
+import requests
 from elasticsearch_dsl import Search
 from flask import request
 
 from .. import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
-from .. import dss_handler, get_logger
+from .. import dss_handler, get_logger, DSSException
 from ..util.es import ElasticsearchClient
 
 
 @dss_handler
 def find():
-    get_logger().debug("Searching for: %s", request.values["query"])
-    # TODO (mbaumann) Use a connection manager
-    es_client = ElasticsearchClient.get(get_logger())
-    query = json.loads(request.values["query"])
-    response = Search(using=es_client,
-                      index=DSS_ELASTICSEARCH_INDEX_NAME,
-                      doc_type=DSS_ELASTICSEARCH_DOC_TYPE).query("match", **query).execute()
-    return {"query": query, "results": format_results(request, response)}
+    raise DSSException(requests.codes.not_implemented,
+                       "simple_search_not_implemented",
+                       "Simple search is not implemented. Use POST /search with Elasticsearch query.")
```

Comment:
OK, thanks for the clarification. I'll remove the GET /search route all together.